app/models/form_handler.rb

```diff
@@ -49,6 +49,10 @@ def current_collection_start_year
     today < window_end_date ? today.year - 1 : today.year
   end
 
+  def current_collection_start_date
+    Time.utc(current_collection_start_year, 4, 5)
```

Comment:
I checked this with the design team, it should be 1st April not 5th

Reply:
Ohh, nice one! Thank you!